src/main/java/Objects/NetworkDevices/NetworkDevice.java

```diff
@@ -10,7 +11,15 @@
     private String id;
     private String team;
     private ArrayList<Integer> connections;
+    private int speed;
+    private Boolean hidden;
+    private int max_packet;
+    private List<String> packets;
 
+    /**
+     *
```

Comment:
you didnt remove my example java doc retard

Reply:
It's because I'm still using it, you retard.